presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -485,6 +485,7 @@ private static ColumnStatistics toColumnStatistics(HiveWriterVersion hiveWriterV
                 null,
                 null,
                 statistics.hasBinaryStatistics() ? toBinaryStatistics(statistics.getBinaryStatistics()) : null,
+                null,
```

Comment:
I'll update DwrfMetadataReader/DwrfMetadataWriter in a separate PR.

Reply:
This metadata is only set the file level and never at the row group level. FYI.